packages/core/contracts/financial-templates/perpetual-multiparty/PerpetualLiquidatable.sol

```diff
@@ -58,6 +58,7 @@ contract PerpetualLiquidatable is PerpetualPositionManager {
         uint256 withdrawalLiveness;
         address collateralAddress;
         address finderAddress;
+        address fpFinderAddress;
```

Comment:
only changes not to liquidatable are to pass in the params. No tests done on Liquidatable; this is left for a separate PR.